src/main/java/com/redhat/rhjmc/containerjfr/documentation_messages/DocumentationMessageModule.java

```diff
@@ -0,0 +1,17 @@
+package com.redhat.rhjmc.containerjfr.documentation_messages;
```

Comment:
Using an underscore in a package name is a bit unconventional. I would prefer `documentationmessages`, or ideally something a bit more concise. IMO it would make sense to lump this in to a `localization` package or something like that.

Reply:
Should `DocumentationManager` and `DocumentationManagerModule` change to `LocalizationManager` and `LocalizationModule` respectively?

Reply:
And should I update the name in the responding container-jfr-web pr?

Reply:
Yea, I think that all makes sense.